web/helpers/authentication_helpers.ex

```diff
@@ -28,16 +28,21 @@ defmodule CodeCorps.AuthenticationHelpers do
   # This is partially adjusted code, taken from canary
   def load_and_authorize_changeset(conn, options) do
     action = conn.private.phoenix_action
-
     cond do
-      action in options[:only] -> conn |> load_resource(options) |> do_init_and_authorize_changeset(options[:model], action)
-      true -> conn
+      action in options[:only] ->
+        conn
+        |> load_resource(options)
+        |> do_init_and_authorize_changeset(options[:model], action)
+      true ->
+        conn
     end
   end
 
   defp do_init_and_authorize_changeset(conn, model, action) do
     changeset = init_changeset(conn, model, action)
-    conn |> assign(:changeset, changeset) |> authorize(changeset, action)
+    conn
+    |> assign(:changeset, changeset)
+    |> authorize(changeset, action)
```

Comment:
I'm not really sure how to resolve that, then. 

We could split up `load_and_authorize_changeset` into `load_changeset` and `authorize_changeset`. Canary already does the same with `_resource_`. For that specific action, since it's a `create`, we could load manually, using a locally defined plug method, then authorize using just `authorize_changeset`